considerations.mkd

```diff
@@ -58,3 +58,38 @@ precision than necessary.
    represents edges as great circles will have different interpretations
    of edges, which vary more the longer the edges are. Try to
    avoid edges of more than 180 degrees in length as far as possible.
+
+
+# Extensibility
+
+In GeoJSON, an "extension" is any property not defined in section 2.
+Consuming implementations that encounter unfamiliar extensions MUST NOT
+stop processing or signal an error and MUST continue processing the
+items as if those properties were not present. Note that support for
```

Comment:
> It is frequently the case that changes to protocols are required
> after they have been put in production.  Protocols that allow the
> introduction of new protocol elements in a way that does not disrupt
> the operation of existing software have proven advantageous in
> practice.

That paragraph that you reference quite specifically mentions "changes to [this] protocol", ie version 2, and quite specifically **does not** mention extensions. In fact, the word extension **does not appear** anywhere in the document.